source/octf/proto/InterfaceTraceManagement.proto

```diff
@@ -40,7 +55,7 @@ service InterfaceTraceManagement {
     };
 
     // Removes specified traces.
-    rpc removeTraces(TracePathPrefix) returns (TraceList) {
+    rpc removeTraces(RemoveTracesRequest) returns (TraceList) {
         option (opts_command).cli = true;
 
         option (opts_command).cli_short_key = "R";
```

Comment:
Maybe we should rename long key to 'remove-traces' since we use plural form in 'list-traces' and in code.